src/Microsoft.Azure.WebJobs.Extensions.Kafka/Microsoft.Azure.WebJobs.Extensions.Kafka.csproj

```diff
@@ -33,10 +33,10 @@
     <WarningsAsErrors />
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Confluent.Kafka" Version="1.6.3" />
-    <PackageReference Include="Confluent.SchemaRegistry" Version="1.6.3" />
-    <PackageReference Include="Confluent.SchemaRegistry.Serdes.Avro" Version="1.6.3" />
-    <PackageReference Include="Confluent.SchemaRegistry.Serdes.Protobuf" Version="1.6.3" />
+    <PackageReference Include="Confluent.Kafka" Version="1.9.0" />
+    <PackageReference Include="Confluent.SchemaRegistry" Version="1.9.0" />
+    <PackageReference Include="Confluent.SchemaRegistry.Serdes.Avro" Version="1.9.0" />
+    <PackageReference Include="Confluent.SchemaRegistry.Serdes.Protobuf" Version="1.9.0" />
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.32" />
```

Comment:
@krishna-kariya  Why are we not upgrading to 1.9.3 version ?

Reply:
@lpapudippu In version [1.9.3](https://github.com/confluentinc/confluent-kafka-dotnet/releases/tag/v1.9.3), there are some changes related to Avro and Protobuf. 
If we want to upgrade to 1.9.3, we would have to check Avro-based Functions (if they are working fine) and even we would have to run the perf experiment on this version.

Reply:
Earlier, all testing was done on 1.9.0 and version 1.9.3 was not released.